src/index.js

```diff
@@ -40,6 +41,7 @@ const fields = [
   { label: "Material", field: "hasMaterialCategory", type: "list-facet", facetSort: "count" },
   { label: "Specimen", field: "hasSpecimenCategory", type: "list-facet", facetSort: "count" },
   { label: "Registrant", field: "registrant", type: "list-facet", facetSort: "count" },
+  { label: "Collection Date", field: "producedBy_resultTimeRange", type: "date-range-facet", minValue: 1800, maxValue: 2022 }
```

Comment:
This is how we declare the new field.  Note the hardcoded min/max.  Those will be replaced by (as yet, unwritten) calls to solr to get the min and max year from the index.